boa_engine/src/syntax/parser/cursor/mod.rs

```diff
@@ -263,15 +263,15 @@ where
 
     /// Check if the peeked token is a line terminator.
     #[inline]
-    pub(super) fn peek_expect_is_line_terminator(
+    pub(super) fn peek_is_line_terminator(
         &mut self,
         skip_n: usize,
         interner: &mut Interner,
-    ) -> Result<bool, ParseError> {
+    ) -> Result<Option<bool>, ParseError> {
```

Comment:
Since this no longer returns an error, we can remove the `Result`, right?

Reply:
We still call `BufferedLexer::peek` and that may return a `ParseError` other than `AbruptEnd`.

Reply:
True, didn't see it!